### PR TITLE
Event export incremental DSL

### PR DESCRIPTION
**Pull request by HassanJaveed84**

### CODE REVIEW

File: dataeng/resources/event-export-incremental.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+echo ${WORKSPACE}/data-czar-keys/${EXPORTER_CONFIG}
+
+aws s3 cp ${WORKSPACE}/data-czar-keys/${EXPORTER_CONFIG} ${EXPORTER_CONFIG_PATH}
+aws s3 cp --recursive ${WORKSPACE}/data-czar-keys/ ${GPG_KEYS_PATH}
+
+####################################################################
+# Run exporter task
+
+if [ "x$NUM_REDUCE_TASKS" = "x" ]; then
```
Comment (pwnage101) [inline dataeng/resources/event-export-incremental.sh]: Also update this script with `[ -z "$NUM_REDUCE_TASKS" ]` and `${WORKSPACE}/...` like the rest.

File: dataeng/jobs/analytics/EventExportIncremental.groovy [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+class EventExportIncremental {
+    public static def job = { dslFactory, allVars ->
+        dslFactory.job("event-export-incremental") {
+            logRotator common_log_rotator(allVars)
+            parameters common_parameters(allVars)
+            parameters from_date_interval_parameter(allVars)
+            parameters to_date_interval_parameter(allVars)
+            parameters {
+                stringParam('OUTPUT_ROOT', allVars.get('OUTPUT_ROOT'))
+                stringParam('EXPORTER_CONFIG', 'config.yaml', 'Exporter configuration relative analytics-secure/analytics-exporter')
```
Comment (pwnage101) [inline dataeng/jobs/analytics/EventExportIncremental.groovy]: nit: "Exporter configuration relative to analytics-secure/analytics-exporter"

File: src/main/groovy/org/edx/jenkins/dsl/AnalyticsConstants.groovy [hunk 3]
```diff
@@ -38,6 +38,32 @@ class AnalyticsConstants {
         }
     }
 
+    public static def additional_multiscm = { allVars ->
```
Comment (pwnage101) [inline src/main/groovy/org/edx/jenkins/dsl/AnalyticsConstants.groovy]: Are these used in any other job?  If not, maybe this should be defined in EventExportIncremental.groovy

**Comment by HassanJaveed84** [comment]
Ready for a final review.

**Pull request merged**